tools/ansible/README.md

```diff
@@ -17,8 +17,8 @@ pip install selinux ansible==2.9 pywinrm
 - A `hosts` file with the required variables defined. See below for an example:
 ```
 [win]
-# Address of a node and its Windows password
-<node_ip> ansible_password=<password>
+# Address of a node and its Windows password and private ip 
+<node_ip> ansible_password=<password> private_ip=<private_ip>
```

Comment:
`Public IP of the node and ...`
Rather than `<node_ip>` let us say say `<public_ip>`

Reply:
Done